refinery/tool_manager/tests.py

```diff
@@ -2778,6 +2778,7 @@ def _start_visualization(
             )
             force_authenticate(self.post_request, self.user)
             post_response = self.tools_view(self.post_request)
+            logger.debug("RESPONSE_DATA: %s", post_response.content)
```

Comment:
Any reason `RESPONSE_DATA` is all caps?

Reply:
@hackdna Not particularly, can change if you'd like

Reply:
Yes, I think it's a good idea to keep logging uniform.